Backend/app/dashboard/Lib/src/Domain/Types/Transaction.hs

```diff
@@ -100,7 +100,7 @@ data Endpoint
   = RideAPI Common.RideEndpoint
   | DriverAPI Common.DriverEndpoint
   | DriverRegistrationAPI Common.DriverRegistrationEndpoint
-  | CustomerAPI Common.CustomerEndpoint
+  | CustomerAPI Common.CustomerEndpointDSL
```

Comment:
you can remove this line because `CustomerEndpointDSL` already added here automatically https://github.com/nammayatri/nammayatri/pull/8867/files#diff-e5a6eae0eede43ba4ddb99f143c64aea6da45abc2b0414959ae29f3737b1e240R20